app/src/main/java/ar/com/wolox/android/cookbook/notifications/di/NotificationScope.kt

```diff
@@ -0,0 +1,7 @@
+package ar.com.wolox.android.cookbook.notifications.di
+
+import javax.inject.Scope
+
+@Scope
+@Retention(AnnotationRetention.RUNTIME)
+annotation class NotificationScope
```

Comment:
Did you create a `Scope` for a particular reason?

Reply:
I did it based on the DataSync recipe, I thought it was for new convention, should I remove it?

Reply:
Nah, it's OK.